packages/lib/src/styles/result-chart.css

```diff
@@ -1,24 +1,36 @@
-
 lens-chart {
-    background-color: var(--white);
-    display: grid;
-  }
-  
-  lens-chart::part(chart-wrapper) {
-    display: grid;
-    grid-template-rows: auto 1fr auto;
-    position: relative;
-  
-  }
-  
-
-  lens-chart::part(chart-title) {
-    text-align: center;
-    margin: 0;
-    padding-bottom: var(--gap-m);
-  }
-  
-  
+  background-color: var(--white);
+  display: grid;
+}
+
+lens-chart::part(chart-wrapper) {
+  display: grid;
+  grid-template-rows: auto 1fr auto;
+  position: relative;
+}
+
+lens-chart::part(chart-overlay) {
```

Comment:
For some reason it is not in the center. Could you please check it

Reply:
It's at the center of the containing card, which also has the advantage that it's center aligned with the chart title. Centering it on the canvas would make things more difficult because then the canvas would have to be a child of the overlay element (you can't put elements inside a canvas afaik). That in turn would mean that the overlay would always have to be present, not only when the text would show. And that would mean that we have to allow mouse events to go through if text is not showing. Overall I don't think it's worth it. Plus it probably wouldn't look centered in the way you want it either because the charts y axis will make it look lopsided even if it is centered on the canvas.